admin/src/components/Navbar.test.jsx

```
@@ -0,0 +1,71 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { vi, describe, beforeEach, test, expect } from "vitest";
+import Navbar from "./Navbar";
+import ChangePassword from "./ChangePassword";
+
+// Mock the ChangePassword component since we don't need to test it here
+vi.mock("./ChangePassword", () => ({
+  __esModule: true,
+  default: ({ isOpen, onClose }) =>
+    isOpen ? (
+      <div role="dialog">
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<dialog>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)